tests/e2e/utils.go

```diff
@@ -90,3 +97,21 @@ func getActiveApplication() string {
 	odoActiveApp := strings.Split(reActiveApp.FindString(stdOut), "*")[1]
 	return strings.TrimSpace(odoActiveApp)
 }
+
+func retryingForOutputMatchStringOfHTTPResponse(url, match string, retry, sleep int) bool {
```

Comment:
Isn't this the same function as in https://github.com/redhat-developer/odo/pull/1401/files#diff-3ee295477bcffbfddd29e7f409916eb2R101 ?

Reply:
Yes